src/components/UserSelector.tsx

```diff
@@ -1,16 +1,69 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { User } from '../types/User';
+import cn from 'classnames';
+
+type Props = {
+  users: User[];
+  currUser: User | null;
+  setCurrUser: (currUser: User) => void;
+};
+
+export const UserSelector: React.FC<Props> = ({
+  users,
+  currUser,
+  setCurrUser = () => {},
```

Comment:
Setting a default value for `setCurrUser` can lead to unexpected behavior if the parent component does not pass this prop explicitly. Consider removing the default value to ensure that the parent component always provides this function.